packages/app/app/components/LibraryView/index.js

```diff
@@ -1,35 +1,56 @@
-import React, { useCallback } from 'react';
-import PropTypes from 'prop-types';
+import React, { useCallback, useMemo } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
```

Comment:
State subscriptions and action creator bindings should be kept in the corresponding container, and passed down as needed. Here they're split between the container and the component.

Reply:
Right; the action creators and state subscriptions should be moved to the container for consistency.

I got a bit confused, subconsciously thinking the `useSelector` calls were needed in `LibraryView`, since I extracted them from the `useMemo` calls (which do need to be in `LibraryView`). I'll move them to the container.

Reply:
Fixed.